packages/java-edition/test/mcfunction/parser/argument.spec.ts

```diff
@@ -290,26 +305,32 @@ const RemoveExtraChildren = new Set([
 	'minecraft:score_holder',
 ])
 
-const project = mockProjectData()
-
-json.initialize(project)
-nbt.initialize(project)
-
-const { meta } = project
-
 describe('mcfunction argument parser', () => {
 	for (const [parserName, cases] of Object.entries(Suites)) {
 		describe(parserName, () => {
-			for (const { content, properties } of cases) {
+			for (const { content, properties, version } of cases) {
 				const treeNode: ArgumentTreeNode = {
 					type: 'argument',
 					parser: parserName as any,
 					properties,
 				}
+				const ctx = typeof version !== 'undefined'
+					? { loadedVersion: version }
+					: void 0
+				const project = mockProjectData({ ctx })
+				json.initialize(project)
+				nbt.initialize(project)
```

Comment:
Does this not make the tests significantly slower?

Reply:
i just timed this whole describe block before/after this PR to see (sample size ~= 5 times):

| before | after | 
|-|-|
| 340-360ms | 330-350ms |

sooo definitely not 😅

Reply:
is there a specific reason you thought initializing stuff for each test would run significantly slower?